lib/components/Editor/CustomExtensions/Markdown/useMarkdownEditor.js

```diff
@@ -68,12 +84,26 @@ const useMarkdownEditor = ({ content, onUpdate, onSubmit }) => {
     inputFieldRef.current.textContent = editorContentRef.current;
   }, [inputFieldRef, editorContentRef]);
 
+  // subscribe to onFocus and onBlur handlers
+  useEffect(() => {
+    if (!inputFieldRef.current) return;
+    inputFieldRef.current.addEventListener("focus", handleFocus);
```

Comment:
Instead of adding the listeners here, why don't we add this props directly to the Markdown editor `textarea` component?